lib/Cake/Console/ShellDispatcher.php

```diff
@@ -140,8 +140,11 @@ protected function _bootstrap() {
 			define('TMP', CAKE_CORE_INCLUDE_PATH . DS . 'Cake' . DS . 'Console' . DS . 'Templates' . DS . 'skel' . DS . 'tmp' . DS);
 		}
 
+		if (!defined('CONFIG')) {
+			define('CONFIG', ROOT . DS . APP_DIR . DS . 'Config' . DS);
+		}
 		// $boot is used by Cake/bootstrap.php file
-		$boot = file_exists(ROOT . DS . APP_DIR . DS . 'Config' . DS . 'bootstrap.php');
+		$boot = file_exists(CONFIG . 'bootstrap.php');
```

Comment:
I just saw https://github.com/cakephp/cakephp/pull/9803. Perhaps a better solution would be to move `$boot` variable to the file where it is actually used, i.e. to Cake/bootstrap.php. @chinpei215, @dereuromark, what would you say?